package/submariner-route-agent.sh

```diff
@@ -9,4 +9,20 @@ else
     DEBUG="-v=4"
 fi
 
-exec submariner-route-agent ${DEBUG} -alsologtostderr
\ No newline at end of file
+# if host is mounted on /host and host has it's own iptables version
+# use that one instead via the shipped iptables wrapper, we do this
+# to avoid configuring iptables and nftables on the host which
+# could lead to functional failures because some hosts use an iptables
+# and iptables-save which program converted nftables instead.
```

Comment:
Don't have a strong opinion on this, but if you plan to update the patch, you can consider this.
"which program converted nftables instead" -> "which program nftables under the hood"

Thanks for the fix @mangelajo :+1:

Reply:
Easy to change, done.